.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -14,17 +14,14 @@ concurrency:
 jobs:
   install-dependencies:
     name: Install
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [macos-latest, ubuntu-latest]
+    runs-on: ubuntu-latest
```

Comment:
The MacOS runners have slightly faster single-core performance.  But they appear to mainly be faster because, unlike Ubuntu, installing Chromium doesn't require the installation of a bunch of [system dependencies](https://github.com/CrowdStrike/glide-core/actions/runs/13768566554/job/38501468784).